Added editor background, workbench colors and integrated terminal color scheme

3ximus opened the pull request:
Hi, I really like your theme, and its the best port of gruvbox to vscode that I have seen. I expanded it to add the theme to more items in the inteface.

I'll will maybe port python syntax colors from vim aswell if i have time for it.

How it looks right now: 
![2017-08-11-175749_1880x1040_scrot](https://user-images.githubusercontent.com/9083012/29223706-72261ec2-7ebf-11e7-99aa-f023e8385685.png)

jdinhlife commented on themes/gruvbox-dark-medium.json:
```diff
@@ -1,6 +1,62 @@
+        "terminal.ansiCyan": "#7db669",
+        "terminal.ansiBrightCyan": "#578e57",
+        "terminal.ansiWhite": "#faefbb",
+        "terminal.ansiBrightWhite": "#e6d4a3",
```
These colors are not in gruvbox palette, where did you get them from? If there's no special reason, it'd be good to stick to the palette as much as possible.

Also, I think it makes more sense to reverse the colors, since `bright` colors should be brighter than the original ones, not darker. Eg.

```
"terminal.ansiRed": "#f73028",
"terminal.ansiBrightRed": "#be0f17",
```

should be 

```
"terminal.ansiRed": "#be0f17",
"terminal.ansiBrightRed": "#f73028",
```

jdinhlife commented on themes/gruvbox-dark-medium.json:
```diff
@@ -1,6 +1,62 @@
+  },
+  "tokenColors":[
     {
       "settings": {
```
I noticed with the `tokenColors` change, this global `settings` block isn't affecting anything anymore, could you please remove this block.

One thing missing from removing this block is `lineHighlight`, so could you also add `"editor.lineHighlightBackground": "#32302f"` to the new global colors setting.

jdinhlife commented on themes/gruvbox-dark-medium.json:
```diff
@@ -1,6 +1,62 @@
+        "activityBar.foreground": "#ebdbb2",
+        "activityBarBadge.background": "#458588",
+        "activityBarBadge.foreground": "#ebdbb2",
+        "sideBar.background": "#282828",
```
Please make these indented to 4 spaces instead (they're at 8 atm whist the upper level is at 2 only). 
Or you can make the whole file having indent size of `4` and run a `Format Document` command in Code. 
Either way are fine, but they should be consistent, not mixed.

3ximus commented:
I added the changes you asked for.
The colors from the terminal were from a konsole gruvbox coloscheme I downloaded from somewhere, I was to lazy do do my own xD . I thought they were the correct ones and didnt bother to check since they were very similar...
I also switched the darker and bright colors however I don't think i like them that way, but they can be easily switched in the settings so I left them the way you asked.

jdinhlife commented:
Thanks @3ximus,

I'll merge this and will add screenshot.
